plugins/discovery-gce/qa/gce/build.gradle

```diff
@@ -39,8 +39,8 @@ restResources {
 }
 
 /** A task to start the GCEFixture which emulates a GCE service **/
-task gceFixture(type: AntFixture) {
-  dependsOn project.sourceSets.yamlRestTest.runtimeClasspath
+tasks.register("gceFixture", AntFixture) {
+  dependsOn "compileTestJava"
```

Comment:
Why was this dependency changed?

Reply:
shouldn't be there, probably a merge issue I ran into today

Reply:
fixed